sim/make/fpga.mk

```diff
@@ -43,6 +43,7 @@ $(fpga_driver_dir)/$(BASE_FILE_NAME)%: $(simulator_verilog) $(fpga_work_dir)/sta
 	cp -f $(GENERATED_DIR)/$(@F) $@
 
 $(fpga_driver_dir)/$(DESIGN)-$(PLATFORM): $($(PLATFORM))
+	mkdir -p $(@D)
 	cp -f $< $@
```

Comment:
```make
$(fpga_build_dir) $(fpga_driver_dir) $(verif_dir) $(repo_state):
        mkdir -p $@

$(fpga_driver_dir)/$(DESIGN)-$(PLATFORM): $($(PLATFORM)) | $(fpga_driver_dir)
	cp -f $< $@
```

This can be avoided by factoring the directory into it's on order-only dep.  What you've done is fine to make forward progress but it would probably be better to factor the `mkdir` out into a pattern rule with clear deps defined.